Makefile

```diff
@@ -88,10 +88,15 @@ compile_commands.json: $(SOURCES)
 	CC=$(CC) CFLAGS="$(CFLAGS) $(EXTRA_CFLAGS)" scripts/gen_compile_commands_json.sh $(SOURCEDIR) > $@
 
 
-.PHONY: clean
+.PHONY: clean, run
```

Comment:
https://unix.stackexchange.com/questions/217295/phony-all-rules-in-gnu-make-file

This should be `.PHONY: clean`